app/utils/posthogUtils.js

```diff
@@ -0,0 +1,21 @@
+import { set } from 'lodash';
+import PostHog from 'posthog-react-native';
+import { POSTHOG_KEY } from '@env';
+const posthog = {
+  client: null
+};
+
+export const getPostHogClient = () => {
+  if (!posthog.client) {
+    set(
+      posthog,
+      'client',
+      new PostHog(POSTHOG_KEY, {
+        // usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+
+        host: 'https://us.i.posthog.com'
```

Comment:
Can we please remove this and add comment saying this is where the custom url goes